package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.1.0",
   "author": "Simon Ihmig <simon.ihmig@kaliber5.de",
   "license": "MIT",
+  "repository": "kaliber5/ember-bootstrap-codemods",
```

Comment:
Not sure: the full URL, or at least the `github:` prefix is not required? See https://docs.npmjs.com/files/package.json#repository

Reply:
You are right. GitHub prefix is required. Otherwise it would be interpreted as a NPM link.

```suggestion
  "repository": "github:kaliber5/ember-bootstrap-codemods",
```